.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+Changes proposed in this PR:
+-
+-
+
+How I've tested this PR:
+
+How I expect reviewers to test this PR:
+
+Checklist:
+- [ ] Tests added
+- [ ] CHANGELOG entry added 
+
+  * HashiCorp engineers only, community PRs should not add a changelog entry.
```

Comment:
Are wanting these lines in the final PR description. If they are meant to be removed, I think we can do something like this.
```suggestion
[//]: # (  * HashiCorp engineers only, community PRs should not add a changelog entry. )
```

Reply:
Nice! This seemed to work for a multiline comment in the markdown:

```
[//]: # ( 
   HashiCorp engineers only. Community PRs should not add a changelog entry.
   Changelog entries should use present tense, e.g. "Add support for..."
)
```

Reply:
The comment is supposed to be so that community contribs don't try and add a changelog entry. So it needs to somehow be associated to that checkbox imo

Reply:
> it needs to somehow be associated to that checkbox imo

Experimented a little (based on [this SO question](https://stackoverflow.com/a/20885980)) and the following seems to work.

```
- [ ] CHANGELOG entry added 

    [HashiCorp engineers only. Community PRs should not add a changelog entry.]::
    [Changelog entries should use present tense, e.g. "Add support for..."]::
```

You do need a blank line prior to the `[comment]::` lines. It also affects the vertical spacing of the bullet points...but it does exclude those comments from rendering.